test/unit/node_test.rb

```diff
@@ -235,4 +235,13 @@ class NodeTest < ActiveSupport::TestCase
     assert !replaced
     assert_equal 'Jingle Jingle Bells', node.body
   end
+
+  test "question has an accepted answer" do
+    question2 = node(:question2)
+    assert_not question2.accepted_answers
+
+    question = node(:question)
+    assert question.accepted_answers
```

Comment:
Likewise here, it probably won't return `true` but you could measure the length with `assert_equal 1, ______` for example.

Hope that helps!